functions/public/Install-omsAgent.ps1

```diff
@@ -26,23 +26,29 @@ function Install-OmsAgent
 		[Alias('IPAddress', 'Name')]
 		[string[]]
 		$computerName = $env:COMPUTERNAME,
-		[Parameter(Mandatory=$true, ParameterSetName='workSpaceClearText')]
+		[Parameter(Mandatory=$true, ParameterSetName='localoms-workSpaceClearText')]
```

Comment:
Can all the ParameterSetName's be changed to camel case so they match what is used throughout the rest of the module.